lib/Models/RegionMapping.js

```diff
@@ -578,6 +578,22 @@ function createNewRegionImageryLayer(regionMapping, layerIndex, regionIndices, g
 
     globeOrMap = defaultValue(globeOrMap, catalogItem.terria.currentViewer);
 
+    var terria = globeOrMap.terria;
+
+    // Inform the user that region mapping will not be supported in old browsers in the future.
+    if (!terria.configParameters.regionMappingOldBrowserNotifiedThisSession && typeof ArrayBuffer === 'undefined') {
+        terria.configParameters.regionMappingOldBrowserNotifiedThisSession = true;
```

Comment:
This is probably over the top, but ideally we'd add this configParameter into Terria.js's `defaultConfigParameters` too, right?